Dockerfile-py2.7

```diff
@@ -48,11 +37,23 @@ ENV QT_TESTING true
 # Xvfb
 ENV DISPLAY :99
 
+# Prevent error: "Qt: Failed to create XKB context!"
+# https://lists.debian.org/debian-backports/2014/10/msg00061.html
+ENV QT_XKB_CONFIG_ROOT=/usr/share/X11/xkb
+
 # Warnings are exceptions
-ENV PYTHONPATH=/Shiboken-1.2.2/shiboken_package/Shiboken
 ENV PYTHONWARNINGS="ignore"
 
 WORKDIR /workspace/Qt.py
+
+RUN cp -avr /usr/lib64/python2.7/site-packages/shiboken* /usr/local/python/lib/python2.7/site-packages && \
+    cp -avr /usr/lib64/python2.7/site-packages/PySide /usr/local/python/lib/python2.7/site-packages
+
+# Experimental (could break things)
```

Comment:
Given we use two different Docker images for Python 2/3, where Python is installed in the very same path - this could work.